src/index.ts

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env node
 
-const { execSync } = require("child_process");
-const path = require("path");
-const Discord = require("discord.js");
-const fs = require("fs-extra");
-const prompts = require("prompts");
-const validatePackageName = require("validate-npm-package-name");
+import { execSync } from "child_process";
+import { Client } from "discord.js";
+import fs from "fs-extra";
+import path from "path";
+import prompts from "prompts";
 
-const appDirectory = path.join(__dirname, "app");
-const appPackage = require(path.join(appDirectory, "package.json"));
-const appToken = { token: "DISCORD_BOT_TOKEN_PLACEHOLDER" };
+import interfaces from "./declarations/interfaces";
+import types from "./declarations/types";
 
-const utilityPackage = require(path.join(__dirname, "package.json"));
-const utilityNameAndVersion = `${utilityPackage.name} v${utilityPackage.version}`;
+const validatePackageName: interfaces.ValidationFunction = require("validate-npm-package-name");
+
+const appDirectory: string = "app";
```

Comment:
I am not that good with paths and all but since the utility script files were moved to the `src` folder, with that it would think that the app folder is `src/app` and if I just do `app` then it would work.

Reply:
ahh, then i think `path.join(__dirname, "../app");` should do it